packages/tslint/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@calcom/tslint",
```

Comment:
Maybe? To avoid confusion with [TypeScript Lint](https://github.com/palantir/tslint)
```suggestion
  "name": "@calcom/localelint",
```

Reply:
Sounds good 👍 I renamed this line as well as the directory's name